### PR TITLE
Regions

### PR DESCRIPTION
**Pull request by MattMacKinnon50**

### CODE REVIEW

**Comment by nwalberts** [comment]
This PR is okay, but Im unsure if it is complete. On a region show page, should I see numerous cryptids for that region? If so, you will need to continue working on this PR. What is the relationship between a cryptid and a region. Given that you have cryptids built already, you should build them into the rest of your feature.

File: config/routes.rb [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
   devise_for :users
 
-  resources :cryptids
-  resources :categories
-  resources :regions
+  resources :cryptids, except: [:delete]
+
+  resources :categories, only: [:index, :show]do
+    resources :cryptids, except: [:delete]
+  end
```
Comment (MattMacKinnon50) [inline config/routes.rb]: remove index :index route here for categories and regions

Comment (MattMacKinnon50) [reply config/routes.rb]: only show for nested cryptids

File: spec/features/02_user_can_see_a_cryptid_show_page_spec.rb [hunk 2]
```diff
@@ -8,11 +8,15 @@
...
      Cryptid.create!(
       name: "Dragon",
       description: "Breathes fire",
-      user: test_user)
+      user: test_user,
+      region: usa,
+      category: reptile)
   end
 
   scenario 'user visits show page' do
```
Comment (MattMacKinnon50) [inline spec/features/02_user_can_see_a_cryptid_show_page_spec.rb]: I should see more information about cryptids on this page including the description and the region and the categories

**Pull request merged**